microsoft-edge/webview2/howto/chromium-devtools-protocol.md

```diff
@@ -0,0 +1,190 @@
+---
+description: Learn how to use the Chrome DevTools Protocol in your WebView2 apps using the Microsoft Edge WebView2 Chromium DevTools Protocol NuGet package
+title: Use Chrome DevTools Protocol in WebView2
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 03/09/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, edge, ICoreWebView2, ICoreWebView2Controller, Chrome DevTools Protocol
+---
+# Use Chromium DevTools Protocol in WebView2  
+
+The [Chromium DevTools Protocol][GitHubChromedevtoolsDevtoolsProtocol] provides the Chrome Debugger Extension API.  The Chrome Debugger Extension API helps you instrument, inspect, debug, and profile Chromium-based browsers.  The Chromium DevTools Protocol is the foundation for the Microsoft Edge \(Chromium\) DevTools.  The WebView team recommends you use the Chromium DevTools Protocol for features that aren't implemented in the WebView2 platform.  For more information about the Chromium DevTools Protocol functionality, navigate to Chromium DevTools Protocol.  
```

Comment:
Is this a little strong recommendation for CDP? We can't guarantee the functionality of any CDP events/methods, that they don't get deprecated/removed, changes to experimental, and so on. It would be nice to tone this down a bit and also somewhere explain the caution